resources/assets/js/misc/datatable/delete.js

```diff
@@ -0,0 +1,34 @@
+exports.deleteItemFromDataTable = (dataTable) => {
```

Comment:
`exports.default = (dataTable) => {` could be used here

A more better aproach would be using es6 exports/imports 
`export default (dataTable) => {` 

`import deleteItemFromDatatable from './delete'`